app/presenters/return-requirements/check.presenter.js

```diff
@@ -32,6 +33,21 @@ function _reasonLink (sessionId, journey) {
   return `/system/return-requirements/${sessionId}/no-returns-required`
 }
 
+function _requirements (session) {
+  const { requirements } = session
+
+  const completedRequirements = []
+
+  for (const requirement of requirements) {
+    const { siteDescription, agreementsExceptions } = requirement
+    if (agreementsExceptions) {
```

Comment:
I'd suggest adding a note for context here.

```suggestion
    // NOTE: We determine a requirement is complete because agreement exceptions is populated and it is the last step in
    // the journey
    if (agreementsExceptions) {
```